File: src/main/java/frc/robot/commands/ArmHoldCommand.java
package frc.robot.commands;

import edu.wpi.first.math.geometry.Rotation2d;
import edu.wpi.first.wpilibj2.command.CommandBase;
import frc.robot.subsystems.arm.Arm;

public class ArmHoldCommand extends CommandBase {
    private final Arm arm;

    public ArmHoldCommand(Arm arm) {
        this.arm = arm;

        addRequirements(arm);
    }

    @Override
    public void execute() {
        arm.runWithSetpoint(arm.getTargetShoulderAngle(), new Rotation2d());
    }
}

File: src/main/java/frc/robot/commands/ArmHoldCommand.kt
package frc.robot.commands

import edu.wpi.first.math.geometry.Rotation2d
import edu.wpi.first.wpilibj2.command.CommandBase
import frc.robot.subsystems.arm.Arm

class ArmHoldCommand(private val arm: Arm) : CommandBase() {
    init {
        addRequirements(arm)
    }

    override fun execute() {
        arm.runWithSetpoint(Arm.State.target.shoulderAngle, Rotation2d())
    }
}